File: Android/app/src/main/java/com/chameleon/tollgate/util/tollgateLog/dto/LogRecord.java
package com.chameleon.tollgate.util.tollgateLog.dto;

import lombok.Getter;

@Getter
public class LogRecord {

	private String timestamp;
	private String level;
	private String factor;
	private String code;
	private String ip;
	
	public LogRecord(String log, String separator){
        String[] logData = log.split(separator);
        timestamp = logData[0];
        ip = logData[1];
        level = logData[2];
        factor = logData[3];
        code = logData[4];
	}
}

File: Client/AuthClient/AuthClient/tollgate/util/tollgateLog/dto/LogRecord.cs
using System;

namespace AuthClient.tollgate.util.tollgateLog.dto
{
    class LogRecord
    {
        public string Timestamp { get; }
        public string Level { get; }
        public string Factor { get; }
        public string Code { get; }
        public string Ip { get; }

        public LogRecord(String log, char separator)
        {
            string[] logData = log.Split(separator);
            Timestamp = logData[0];
            Ip = logData[1];
            Level = logData[2];
            Factor = logData[3];
            Code = logData[4];
        }
    }
}
